backend/src/main/java/org/sefglobal/core/repository/AmbassadorRepository.java

```diff
@@ -10,4 +12,5 @@
 public interface AmbassadorRepository extends JpaRepository<Ambassador, Long> {
     Ambassador findByIdAndStatus(long id, String status);
     List<Ambassador> findAllByStatus(String status);
+    Page<Ambassador> findAll(Pageable pageable);
```

Comment:
I think it's better if we could use another method name instead of overloading the findAll method.

Reply:
what about `findAllBy`

Reply:
Instead of defining methods by params that are passed to them just overload these methods with the name `findAll`

Reply:
@YohanAvishke what if we need the default `findAll` method in a different scenario?

Reply:
Makes sense @jayasanka-sack

Reply:
@anjula-sack Change the method name

Reply:
I changed it to `findAmbassadorsBy`

Reply:
Let's add a meaningful name, 
ex: getPaginatedAmbassadors